client/my-sites/customer-home/controller.jsx

```diff
@@ -41,10 +46,15 @@ export async function maybeRedirect( context, next ) {
 
 	const siteId = getSelectedSiteId( state );
 	const site = getSelectedSite( state );
+	const adminInterface = getSiteOption( state, siteId, 'wpcom_admin_interface' );
```

Comment:
Are these populated in time for the redirect?

Reply:
(I'll test tomorrow)

Reply:
When I tested, the values were populated in time.